obstacle_range: 3.0
raytrace_range: 3.5

footprint: [[-0.105, -0.105], [-0.105, 0.105], [0.041, 0.105], [0.041, -0.105]]
#robot_radius: 0.105

inflation_radius: 0.55
cost_scaling_factor: 2.58

map_type: costmap
observation_sources: point_cloud
point_cloud: {
    sensor_frame: camera_link,
    data_type: PointCloud2,
    topic: /orb_slam3/map_points,
    marking: true,
    clearing: true,
    min_obstacle_height: 0.1,
    max_obstacle_height: 0.5
}